requirements.txt

```diff
@@ -11,11 +11,11 @@ xlsxwriter
 jinja2
 
 # Build/Test Requirements
-cx_freeze==5.0.2
+cx_freeze==6.1.0
 nose
 parameterized
 coverage
 autopep8
-flake8==3.6.0
+flake8==3.7.0
```

Comment:
The build fails on the code style check step. Maybe we should keep the existing version, or disable some rules?